app/iot/service.py

```diff
@@ -1,49 +1,62 @@
+import asyncio
 import random
 import string
+from typing import Any, Awaitable
 from typing import Protocol
 
 from .message import Message, MessageType
```

Comment:
Ensure that the `Message` and `MessageType` are correctly defined in the `message` module. If they are not defined or imported correctly, it will cause errors when trying to use them in the `send_msg` and `run_program` methods.